vonk/features/customresources.rst

```diff
@@ -51,11 +53,8 @@ To test whether you can actually use the endpoint associated with your custom re
 
 Now use your favorite editor to create a resource that conforms to the Foo StructureDefinition. And then create it on Vonk: ``POST <base-url>/Foo``.
 
-All the operations on specification-defined resourcetypes are also available for custom resources. You can also use them in a batch or transaction bundle. There is one limitation however:
-
-* Validation: Custom Resources can not be validated. This implies that :ref:`feature_prevalidation` cannot be used in conjunction with Custom Resources.
-
-We expect to remove this limitation in a future version of Vonk.
+All the operations on specification-defined resourcetypes are also available for custom resources. You can also use them in a batch or transaction bundle. 
+Custom Resources can also be validated. This implies that :ref:`feature_prevalidation` can be used in conjunction with Custom Resources.
```

Comment:
did you mean 'means' instead of 'implies'?

Reply:
'also means'